src/mn_internal.h

```diff
@@ -102,7 +102,7 @@ void MN_DrawEnemy(void);
 #define S_HILITE      0x00000001 // Cursor is sitting on this item
 #define S_SELECT      0x00000002 // We're changing this item
 #define S_TITLE       0x00000004 // Title item
-#define S_FUNC        0x00000008 // Call var.func() on MENU_ENTER
+#define S_CALIBR      0x00000008 // Calibration item
```

Comment:
Thanks for looking at this, but I had some future ideas for `S_FUNC`, that's why I kept it generalized.

Reply:
Okay, how about current version?

Reply:
I think that's a good compromise and cleaner than what I had originally. I haven't tested with ASAN, so I'll go with your judgment.